components/document.css

```diff
@@ -0,0 +1,27 @@
+@import '../../homepage/homepage.css';
```

Comment:
Alright, I suppose this is the css file for each component's pages
So, import `homepage.css` here and don't import again in other components